From ebde1f7390ca4d2a9b10fa650521cbbed3aa34c3 Mon Sep 17 00:00:00 2001
From: Lars Gierth <larsg@systemli.org>
Date: Mon, 2 Feb 2015 00:41:43 +0100
Subject: [PATCH] build: parse optimizeLevel from CFLAGS

---
 node_build/make.js | 14 ++++++++++++--
 1 file changed, 12 insertions(+), 2 deletions(-)

diff --git a/node_build/make.js b/node_build/make.js
index c048053..8b5f628 100644
--- a/node_build/make.js
+++ b/node_build/make.js
@@ -117,9 +117,19 @@ Builder.configure({
     }
 
     // Install any user-defined CFLAGS. Necessary if you are messing about with building cnacl
-    // with NEON on the BBB
+    // with NEON on the BBB, or want to set -Os (OpenWrt)
     if (CFLAGS) {
-        [].push.apply(builder.config.cflags, CFLAGS.split(' '));
+        var cflags = CFLAGS.split(' ');
+        cflags.forEach(function(flag) {
+             if (/^\-O[^2s]$/.test(flag)) {
+                console.log("Skipping " + flag + ", assuming " +
+                            builder.config.optimizeLevel + " instead.");
+            } else if (/^\-O[2s]$/.test(flag)) {
+                builder.config.optimizeLevel = flag;
+            } else {
+                [].push.apply(builder.config.cflags, cflags);
+            }
+        });
     }
 
     // We also need to pass various architecture/floating point flags to GCC when invoked as
-- 
2.1.0

