diff --git a/node_build/make.js b/node_build/make.js
index 02d8137..199076f 100644
--- a/node_build/make.js
+++ b/node_build/make.js
@@ -167,6 +167,12 @@ Builder.configure({
         );
     }
 
+    if (/uclibc/i.test(GCC)) {
+        builder.config.libs.push(
+            '-lssp'
+        );
+    }
+
     // Build dependencies
     nThen(function (waitFor) {
         Fs.exists(BUILDDIR+'/dependencies', waitFor(function (exists) {
