diff --git a/node_build/make.js b/node_build/make.js
index 56c8ee5..8a096eb 100644
--- a/node_build/make.js
+++ b/node_build/make.js
@@ -62,13 +62,6 @@ Builder.configure({
         '-Wno-unused-parameter',
         '-Wno-unused-result',

-        // Broken GCC patch makes -fstack-protector-all not work
-        // workaround is to give -fno-stack-protector first.
-        // see: https://bugs.launchpad.net/ubuntu/+source/gcc-4.5/+bug/691722
-        '-fno-stack-protector',
-        '-fstack-protector-all',
-        '-Wstack-protector',
-
         '-D','HAS_BUILTIN_CONSTANT_P',

         '-g',
@@ -153,6 +146,20 @@ Builder.configure({
         });
     }

+    uclibc = /uclibc/i.test(GCC);
+    libssp = process.env['SSP_SUPPORT'] == 'y'
+    if (!uclibc || libssp) {
+        builder.config.cflags.push(
+            // Broken GCC patch makes -fstack-protector-all not work
+            // workaround is to give -fno-stack-protector first.
+            // see: https://bugs.launchpad.net/ubuntu/+source/gcc-4.5/+bug/691722
+            '-fno-stack-protector',
+            '-fstack-protector-all',
+            '-Wstack-protector'
+        );
+        if (uclibc) { builder.config.libs.push('-lssp'); }
+    }
+
     // Build dependencies
     nThen(function (waitFor) {
         Fs.exists(BUILDDIR+'/dependencies', waitFor(function (exists) {
