--- a/admin/angel/cjdroute2.c
+++ b/admin/angel/cjdroute2.c
@@ -189,7 +189,6 @@
            "        ]\n");
 #ifdef HAS_ETH_INTERFACE
     printf("\n"
-           "        /*\n"
            "        \"ETHInterface\":\n"
            "        [\n"
            "            {\n"
@@ -221,7 +220,6 @@
            "                }\n"
            "            }\n"
            "        ]\n"
-           "        */\n"
            "\n");
 #endif
     printf("    },\n"

